src/Utils/getUser.ts

```diff
@@ -30,3 +30,7 @@ export function userHasLabFeature(user: User, featureName: string): boolean {
 
   return lab_features.includes(featureName)
 }
+
+export function userIsAdmin(user?: User): boolean {
```

Comment:
I _could_ add tests for this, but it seems pretty simple. If you feel like I should cover this with tests, let me know.

Reply:
Test wouldn't hurt for utility things like this. Would just take a sec and prevent breakage in the future.

Reply:
One other slight refactory-y thing (if you wouldn't mind :) would be to rename this file from `getUser` to just `user`, since it now contains multiple utilities. Originally it was mean to follow 1:1 with the exported file at the top, but has since expanded.

Reply:
Would you rather have it broken into individual files for each function, or do you prefer them all in that one file?

Reply:
I think one file is fine for this

Reply:
All set on these changes! Let me know how they look.